en/components/grids_templates/sorting.md

```diff
@@ -141,6 +141,50 @@ The @@igComponent supports remote sorting, which is demonstrated in the [`@@igCo
 <div class="divider--half"></div>
 }
 
+## Sorting Indicators Templates
+
+@@if (igxName === 'IgxGrid') {
+
+If you want to customize the sorting indicator icon in the column header you could use custom templates. The Grid provides three directives for templating the indicators for all states of the column sorting:
```

Comment:
Try to use more passive voice, instead of `you`, `we` or `I`:

The sorting indicator icon can be template...

Reply:
The following directives are available for templating the sorting indicator for any sorting state (Ascending, descending, none):